src/app/item-page/orcid-page/orcid-sync-settings/orcid-sync-settings.component.ts

```diff
@@ -156,7 +156,8 @@ export class OrcidSyncSettingsComponent implements OnInit {
         }
       }),
     ).subscribe((remoteData: RemoteData<ResearcherProfile>) => {
-      if (remoteData.isSuccess) {
+      // hasSucceeded is true if the response is success or successStale
+      if (remoteData.hasSucceeded) {
```

Comment:
This seems unrelated to this PR.  But, it seems like an OK change.  It'd just be nice to document what this is fixing.  Perhaps add a note to this PR's description as to why we are making this change to the ORCID-sync-settings component?  Or maybe split this into a separate, small PR to document what bug this fixes?